src/templates/_macros/market_button.html

```diff
@@ -9,7 +9,7 @@
         {{ data_attrs|make_data_attrs }}
         {{ 'disabled' if app.disabled }}
         data-product="{{ app|json }}"
-        {% if app.isWebsite %}href="{{ app.url }}"{% endif %}>
+        {% if app.isWebsite %}href="{% if app.mobile_url %}{{ app.mobile_url }}{% else %}{{ app.url }}{% endif %}"{% endif %}>
```

Comment:
I thought about checking `caps.os.type === 'desktop'` too but I figured I'd cross that bridge when we enable desktop websites.